<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QDialog" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 188, 46);</string>
  </property>
  <widget class="QLabel" name="label_signup">
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>240</y>
     <width>290</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>48</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_next">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>480</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
border: 2px solid white;
border-radius:  15px; </string>
   </property>
   <property name="text">
    <string>NEXT</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_account">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>332</y>
     <width>271</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:  10px; 
background-color: rgb(255, 255, 255);

color: rgb(127, 127, 127);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>EMAIL</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="foreground">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>600</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 40);
border-radius:  20px; </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>271</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:  10px; 
background-color: rgb(255, 255, 255);

color: rgb(127, 127, 127);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/back arrow.png);
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>332</y>
     <width>151</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:  10px; 
color: rgb(255, 255, 255);
selection-background-color: rgba(255, 255, 255, 0);
background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>@nycu.edu.tw</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>@gmail.com</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>380</y>
     <width>161</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:  12px; 
background-color: rgb(255, 255, 255,0);
color: rgb(255, 255, 255);
font: 10pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>8~15 characters</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>400</y>
     <width>171</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:  12px; 
background-color: rgb(255, 255, 255,0);
color: rgb(255, 255, 255);
font: 10pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>at least 1 digit</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_alert">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>440</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Consolas&quot;;
color: rgb(64, 64, 64);
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
border: 3px;

</string>
   </property>
  </widget>
  <zorder>foreground</zorder>
  <zorder>label_signup</zorder>
  <zorder>lineEdit_account</zorder>
  <zorder>pushButton_next</zorder>
  <zorder>lineEdit_password</zorder>
  <zorder>pushButton_back</zorder>
  <zorder>comboBox</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>textBrowser_alert</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
